src/lib/turtle.js

```diff
@@ -2375,12 +2429,12 @@ function generateTurtleModule(_target) {
         return _focus;
     }
 
-    function resetTurtle() {
+    function resetTurtle(keepShowingUI) {
```

Comment:
Similarly I'd document this (I know the function isn't documented yet, but we can improve things while we're modifying).